cli/flox/src/commands/auth.rs

```diff
@@ -179,8 +179,8 @@ pub enum Auth {
     Logout,
 
     /// Get current username
```

Comment:
nit consistent with man page
```suggestion
    /// Print your current login status
```